metricshub-hardware/src/main/java/org/sentrysoftware/metricshub/hardware/threshold/AbstractMetricNormalizer.java

```diff
@@ -184,19 +184,19 @@ protected void normalizeErrorsLimitMetric(Monitor monitor) {
 	}
 
 	/**
-	 * Adjusts the values of degraded and critical metrics if necessary.
-	 * If the critical value is smaller than the degraded value, their values are swapped.
+	 * Adjusts the values of two metrics if necessary.
+	 * If the value of the first metric is smaller than the value of the second metric, their values are swapped.
 	 *
-	 * @param criticalMetric The critical metric
-	 * @param degradedMetric The degraded metric
+	 * @param firstMetric  The first metric
+	 * @param secondMetric The second metric
 	 */
-	protected void adjustMetricsIfNecessary(final NumberMetric criticalMetric, final NumberMetric degradedMetric) {
-		final Double degradedValue = degradedMetric.getValue();
-		final Double criticalValue = criticalMetric.getValue();
+	protected void adjustMetricsIfNecessary(final NumberMetric firstMetric, final NumberMetric secondMetric) {

```

Comment:
Rename the method to `swapIfFirstLessThanSecond` to provide clarity and indicate that the method will perform a swap if the first metric's value is less than the second metric's value.